CHANGELOG

```diff
@@ -1,3 +1,9 @@
+2.0.3 2015-03-27
+- Fix `graphite-web` install when using pip and default `root_dir`
+- Restart services when changing package versions
+- Improve acceptance and integration test coverage
+- Remove support for Ruby 1.8.7 and 1.9.2
```

Comment:
Should this be a major version bump given that we're removing support for older Ruby versions?